generators/server/index.js

```diff
@@ -259,6 +264,11 @@ module.exports = JhipsterServerGenerator.extend({
                     this.serviceDiscoveryType = false;
                 }
 
+                // If the service discovery is not defined, it is disabled by default
+                if (this.apiFirst === undefined) {
```

Comment:
wouldn't this be set already above seems redundant and by making it boolean such defaulting will be unnecessary anyway